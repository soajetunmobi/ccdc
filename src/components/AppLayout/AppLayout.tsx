import React, { FC, PropsWithChildren, ReactNode } from 'react'
import { Header } from './Header'
import { Main } from './Main'
import { Footer } from './Footer'

export const AppLayout: FC = ({ children }: PropsWithChildren<ReactNode>) => {
  return (
    <>
      <Header />
      <Main>{children}</Main>
      <Footer />
    </>
  )
}
