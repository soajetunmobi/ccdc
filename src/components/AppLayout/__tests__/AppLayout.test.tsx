import React from 'react'
import { AppLayout } from 'components/AppLayout'
import { render } from '@testing-library/react'

test('it renders AppLayout correctly', () => {
  const { container } = render(<AppLayout />)
  expect(container).toMatchSnapshot()
})
