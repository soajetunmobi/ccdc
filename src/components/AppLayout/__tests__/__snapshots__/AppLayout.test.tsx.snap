// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders AppLayout correctly 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="logo"
    >
      <span
        class="cursor--pointer"
        data-testid="logo-link"
      >
        CCDC
      </span>
    </div>
  </div>
  <main
    class="content"
  />
  <footer
    class="footer"
  >
    <span>
      CCDC
    </span>
    <br />
    2021
  </footer>
</div>
`;
